tipg/settings.py

```diff
@@ -104,6 +103,8 @@ class APISettings(pydantic.BaseSettings):
 
     add_tiles_viewer: bool = True
 
+    catalog_ttl: int = 300
```

Comment:
Change the `catalog_ttl` to be a string so we can easily set it with env variable 

```
TIPG_CATALOG_TTL=10 uvicorn tipg.main:app --port 8080 --reload
```